diff --git a/node_modules/@clerk/shared/dist/react/index.mjs b/node_modules/@clerk/shared/dist/react/index.mjs
index 7a390cb..3826ff6 100644
--- a/node_modules/@clerk/shared/dist/react/index.mjs
+++ b/node_modules/@clerk/shared/dist/react/index.mjs
@@ -57,7 +57,7 @@ __export(clerk_swr_exports, {
 });
 __reExport(clerk_swr_exports, swr_star);
 import * as swr_star from "swr";
-import { default as default2, SWRConfig } from "swr";
+import { SWRConfig } from "swr";
 import { default as default3 } from "swr/infinite";
 
 // src/react/contexts.tsx
